parm/metplus_config/stats/global_ens/atmos_grid2grid/EnsembleStat_fcstNAEFS_obsModelAnalysis_climoERA5.conf

```diff
@@ -220,9 +221,9 @@ ENSEMBLE_STAT_OUTPUT_DIR = {OUTPUT_BASE}/stat/{model}
 #ENSEMBLE_STAT_CLIMO_STDEV_INPUT_DIR = {climpath}
 
 ENSEMBLE_STAT_CLIMO_MEAN_FILE_NAME = {climpath}/mean_{valid?fmt=%m%d}
-ENSEMBLE_STAT_CLIMO_MEAN_USE_FCST = True
+#ENSEMBLE_STAT_CLIMO_MEAN_USE_FCST = True
```

Comment:
@StevenSimon-NOAA Similar to my previous comment, can these commented out lines please be removed from the PR? Thanks!